Paintroid/src/main/res/layout/pocketpaint_item_layer.xml

```diff
@@ -18,20 +19,18 @@
  -->
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
     android:id="@+id/pocketpaint_item_layer_background"
-    android:layout_width="150dp"
-    android:layout_height="wrap_content"
+    android:layout_width="200dp"
+    android:layout_height="170dp"
```

Comment:
sure about fixing size?

Reply:
same as above. The height is necassary because we support to display the preview with the correct aspect ratio. with wrap content this would also compress the opacity seekbar dramatically. Also a fixed height won't cause any problems